docs/_articles/en/v3.x/release-information.md

```diff
@@ -6,6 +6,36 @@ lang: en
 ### To update the plugin, please follow the linked [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in)
 
 
+## [v3.3.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.3.0) (07-20-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.2.0...v3.3.0)
+
+### Release Summary
+* SFGE: Added new entry points for CRUD/FLS rule
```

Comment:
@rmohan20 One of the suggestion from @johnbelosf was to add what these entry points were in the release notes so that the Tech Evangelist team can learn more and work with partners.